package.json

```diff
@@ -19,33 +19,53 @@
     "rulesDirectory": {
       "type": "string",
       "title": "Custom rules directory (absolute path)",
-      "default": ""
+      "default": "",
+      "order": 2
     },
     "useLocalTslint": {
       "type": "boolean",
-      "title": "Try using the local tslint package (if exist)",
-      "default": true
+      "title": "Try to use the project's local tslint package, if it exists",
+      "default": true,
+      "order": 5
     },
     "enableSemanticRules": {
       "type": "boolean",
       "title": "Enable semantic rules",
       "description": "Allow passing a TypeScript program object to the linter. May negatively affect performance. See this page for details: https://palantir.github.io/tslint/usage/type-checking/",
-      "default": false
+      "default": false,
+      "order": 1
     },
     "ignoreTypings": {
       "type": "boolean",
       "title": "Ignore typings files (.d.ts)",
-      "default": false
+      "default": false,
+      "order": 4
     },
     "fixOnSave": {
       "title": "Fix errors on save",
       "description": "Have tslint attempt to fix some errors automatically when saving the file.",
       "type": "boolean",
-      "default": false
+      "default": false,
+      "order": 3
+    },
+    "useGlobalTslint": {
+      "type": "boolean",
+      "title": "Use the global tslint install if there is no local version (or it is disabled)",
```

Comment:
This should be short and sweet: `Use the global TSLint install`. Combine the rest into the description.